submissions/Complexity Corner - The Mall/Complexity Corner - The Mall.md

```diff
@@ -0,0 +1,105 @@
+### Complexity Corner: The Mall <author>The Kool</author>
+
+Welcome back once more to the Complexity Corner.  Today I'll be exploring a technique instead of a technology: the Mall.  As always with these articles, we'll be asking three questions: 'What is it?', 'What results can it give?', and 'Where should it be used?'
+
+## What is this 'Mall' I keep hearing about?
+
+We've all experienced it by now, setting up something to craft manually and then drumming your fingers on the desk while you wait for the mile-long queue to finish.  Wouldn't it be so much nicer to walk over to a chest and have some waiting for you?  To just go shopping for your next project, instead of crafting by hand?  That's what the Mall is for.  Without even realizing it, you probably only think of automating things that have an automated end goal, such as ammo for your turrets or science for your labs.  The Mall is a place where you automate things that have a manual end goal - You.
```

Comment:
Would it benefit from a little history part? iirc, the idea have been popularised by KoS vids, and later on by Nilaus (that is naming this concept differently for whatever reasons). May be nice to find some early examples, maybe from the start of factorio MP games in 0.12?

Reply:
Comment by JD Plays: 
First term 'mall' popularised by KOS, but also falls under other names, I use the term Workshop, Nilaus uses a Hub, others use Complexity Corner, as they are complicated to build, but simplify your factorio experience